BattleShipsClient/BattleShipsClient/Program.cs

```diff
@@ -11,12 +12,36 @@ static class Program
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
+        
         [STAThread]
         static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new Menu());
+            //Application.Run(new Form1());
+        }
+        static string LogName;
+        static object LogLock = new object();
+        public static void Log(string message)
+        {
+            lock (LogLock)
+            {
+                using (StreamWriter swAppend = File.AppendText(LogName))
+                {
+                    swAppend.WriteLine("[" + DateTime.Now.Hour.ToString() + ":" + DateTime.Now.Minute.ToString() + ":" + DateTime.Now.Second.ToString() + "]" + " - " + message);
```

Comment:
```suggestion
                    swAppend.WriteLine($"[{DateTime.Now.Hour}:{DateTime.Now.Minute}:{DateTime.Now.Second}] - {message}");
```
By using string interpolation, you can make it clearer what text